### PR TITLE
Patch windows embedding to latest version

### PR DESCRIPTION
**Pull request by gargakshit**
Fixes high CPU usage on windows

### CODE REVIEW

File: windows/runner/flutter_window.cpp [hunk 1]
```diff
@@ -58,9 +55,6 @@ FlutterWindow::MessageHandler(HWND hwnd, UINT const message,
     case WM_FONTCHANGE:
       flutter_controller_->engine()->ReloadSystemFonts();
       break;
-    case WM_DWMCOLORIZATIONCOLORCHANGED:
-      flutter_controller_->engine()->ReloadPlatformBrightness();
-      break;
```
Comment (dvd-bnc) [inline windows/runner/flutter_window.cpp]: this part was needed for windows system theme, can you confirm it works even without it?

File: windows/runner/win32_window.cpp [hunk 2]
```diff
@@ -173,39 +173,26 @@ Win32Window::MessageHandler(HWND hwnd,
...
         SetFocus(child_content_);
       }
       return 0;
-    case WM_GETMINMAXINFO:
-      auto info = reinterpret_cast<MINMAXINFO *>(lparam);
-      Size minSize = Size(400, 600);
-      getSizeOnScreen(hwnd, &minSize);
-      info->ptMinTrackSize.x = minSize.width;
-      info->ptMinTrackSize.y = minSize.height;
-      return 0;
```
Comment (dvd-bnc) [inline windows/runner/win32_window.cpp:196]: i remember adding this part manually to set the minimum size of the window, idk if it's better to keep it or not tho

File: windows/runner/win32_window.cpp [hunk 3]
```diff
@@ -173,39 +173,26 @@ Win32Window::MessageHandler(HWND hwnd,
...
   return DefWindowProc(window_handle_, message, wparam, lparam);
 }
 
-void Win32Window::getSizeOnScreen(HWND window, Win32Window::Size *size) {
-    UINT dpi = GetDpiForWindow(window);
-    double scale_factor = dpi / 96.0;
-    size->width = static_cast<int>(size->width * scale_factor);
-    size->height = static_cast<int>(size->height * scale_factor);
-}
-
```
Comment (dvd-bnc) [inline windows/runner/win32_window.cpp:208]: same thing as above

File: windows/runner/win32_window.h [hunk 4]
```diff
@@ -93,8 +93,6 @@ class Win32Window {
 
   // window handle for hosted content.
   HWND child_content_ = nullptr;
-
-  void getSizeOnScreen(HWND window, Size *size);
```
Comment (dvd-bnc) [inline windows/runner/win32_window.h:97]: yea

File: windows/runner/win32_window.cpp (hunk 2 above)
Comment (gargakshit) [reply windows/runner/win32_window.cpp:196]: Better to use bitsdojo window's API which works fine

File: windows/runner/flutter_window.cpp (hunk 1 above)
Comment (gargakshit) [reply windows/runner/flutter_window.cpp]: Pushing the fix

Comment (gargakshit) [reply windows/runner/flutter_window.cpp]: Fixed in 86d8e2a960b5bd02a231be4123eed62830b4c2c5

**Comment by dvd-bnc** [comment]
yea good now yea

**Comment by gargakshit** [comment]
Deleting the branch

**Pull request merged**